plugins/package-managers/node/src/main/kotlin/Npm.kt

```diff
@@ -241,7 +241,7 @@ open class Npm(
         }
     }
 
-    internal open fun getRemotePackageDetails(workingDir: File, packageName: String): PackageJson? {
+    protected open fun getRemotePackageDetails(workingDir: File, packageName: String): PackageJson? {
```

Comment:
The commit message does not seem to match the diff. Also, typo in "unsucessful".

Reply:
Dropped the accidental commit.